docs/configuration_options.rst

```diff
@@ -88,6 +88,16 @@ Data Conversion
   * Default: false
   * Importance: low
 
+``compact.map.entries``
+  Defines how map entries with string keys within record values should be written to JSON. When this is set to ``true``, these entries are written compactly as ``\"entryKey\": \"entryValue\"``.
+  Otherwise, map entries with string keys are written as a nested document ``{\"key\": \"entryKey\", \"value\": \"entryValue\"}``.
```

Comment:
Is escaping of `"` intentional in the text too? I guess it is, but I'm checking.

Reply:
yes, it is intentional.